app/experimenter/nimbus-ui/src/components/HeaderExperiment/index.tsx

```diff
@@ -51,42 +33,6 @@ const HeaderExperiment = ({
     </p>
     <div className="row">
       <div className="col">
-        {summaryView && status.draft && (
-          <StatusLink
-            to="edit"
-            label="Edit Experiment"
-            storiesOf="pages/PageEdit"
-            Icon={Cog}
-          />
-        )}
-
-        {summaryView && status.preview && (
-          <StatusLink
-            to="request-review"
-            label="Go to Review"
-            storiesOf="pages/RequestReview"
-            Icon={Clipboard}
-          />
-        )}
```

Comment:
[Added this link as part of the preview UI polish](https://github.com/mozilla/experimenter/commit/4f3a35ce53a0bf30ce5ae6941c1cb47d91b99c6c#diff-c1a6106159bcbf6778aa91c18f751f679859aded655ca34784d7971828ff2f6cR63-R71). Without it, there's no way to get to the form to transition back to DRAFT or forward to REVIEW that lives at `request-review`.

Better option might be to move that `FormLaunchPreviewToReview` subcomponent in `PageRequestReview` over to the summary page? Might make sense to live in the Summary component itself, but we haven't had components other than pages handling API calls, I don't think

<img width="280" alt="Capture" src="https://user-images.githubusercontent.com/21687/108929501-079d7c00-75f9-11eb-903a-41ea289ffedf.PNG">

Reply:
Ohhh, hmm that's right.

> Better option might be to move that `FormLaunchPreviewToReview` subcomponent in `PageRequestReview` over to the summary page?

That or keep "Review & Launch" in the sidebar while in Preview. @AnaMedinac @k88hudson do you have thoughts here? I can reinstate the header link, but it'll mean keeping the `summaryView` wiring in place.

Reply:
I kinda feel like the sidebar while in Preview should use [this style](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/4b149db457b3b8eaedb151d1d1755cf8f37782f6/nimbus-ui/index.html?path=/story/components-applayoutwithsidebar--status-review) anyway, which it does in Review. Currently we go from Edit sidebar (draft), to Locked (preview), to partially-disabled Edit (review), and back to Locked (released). Using that partially-disabled Edit sidebar in Preview would also solve this issue (it would look [like this](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/2a1cd8cba6660894aff588ae695c5fbe06c5562f/nimbus-ui/index.html?path=/story/components-applayoutwithsidebar--status-review))

Reply:
I also personally like the idea of having the partially-disabled sidebar in preview. Also, in case the experiment is rejected I'm assuming the sidebar would go back to enabled and it'd be clear for the user that they can edit it.

Reply:
I don't love the partially disabled sidebar – i'd probably just hide it –but seems totally find to try and see how it goes!

Reply:
Thanks for all the input here! On further discussion and this feedback, we'll use the disabled sidebar, with the titles. Here's what this looks like now from the perspective of the [Request Review](https://storage.googleapis.com/mozilla-storybooks-experimenter/commits/8ab441a900a2182856c916160890df1fcd02016f/nimbus-ui/index.html?path=/story/pages-requestreview--preview-status) page